setup.py

```diff
@@ -23,7 +23,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.0.33',
+    version='0.0.34',
```

Comment:
Please also generate a new wheel and source tar. (those can be done by running `python3 ./setup.py bdist_wheel` and `python3 ./setup.py sdist`).

Thanks!